src/Storages/StoragePostgreSQL.cpp

```diff
@@ -539,6 +539,8 @@ void registerStoragePostgreSQL(StorageFactory & factory)
     {
         auto configuration = StoragePostgreSQL::getConfiguration(args.engine_args, args.getLocalContext());
         const auto & settings = args.getContext()->getSettingsRef();
+        if (!settings.postgresql_connection_pool_size)
```

Comment:
I had an impression that the issue https://github.com/ClickHouse/ClickHouse/issues/52343 means a bit different thing. The issue tells about settings like in MySQL storage engine - these settings are table engine level settings and are different from `DB::Settings`. But in this PR settings which will work are a part of `DB::Settings`, e.g. if we will want to add some settings for postgresql engine we will need to add them to a list of `COMMON_SETTINGS`, while for MySQL they are added to a list `LIST_OF_MYSQL_SETTINGS`. I'd expect similar behaviour for `PostgreSQL`. @alexey-milovidov, wdyt?

Reply:
Yes, let's make it consistent with the MySQL storage engine.